src/AppInstallerCLICore/Workflows/ImportExportFlow.cpp

```diff
@@ -320,12 +320,15 @@ namespace AppInstaller::CLI::Workflow
                     }
                 }
 
-                packagesToInstall.emplace_back(
+                Execution::PackageToInstall package{
                     std::move(searchContext.Get<Execution::Data::PackageVersion>()),
                     std::move(searchContext.Get<Execution::Data::InstalledPackageVersion>()),
                     std::move(searchContext.Get<Execution::Data::Manifest>()),
                     std::move(searchContext.Get<Execution::Data::Installer>().value()),
-                    packageRequest.Scope);
+                    packageRequest.Scope,
+                    subExecution.GetCurrentSubExecutionId()};
+
+                packagesToInstall.emplace_back(std::move(package));

```

Comment:
nit: Why change this?  It isn't really a big deal perf-wise, but previously it was direct construction and now it is construct and move.